packages/legend-studio/src/models/protocols/pure/PureProtocolProcessorPlugin.ts

```diff
@@ -53,4 +53,6 @@ export abstract class PureProtocolProcessorPlugin extends AbstractPlugin {
   V1_getExtraElementTransformers?(): V1_ElementTransformer[];
 
   V1_getExtraSourceInformationKeys?(): string[];
+
+  V1_getExtraMappingExecutionClasses?(): Clazz<V1_PackageableElement>[];
```

Comment:
let's change this to `V1_getExtraExuectionInputElements?(): V1_PackageableElement[]` and add a little bit of doc to this